src/pypi_changes/_cli.py

```diff
@@ -43,6 +44,18 @@ def _define_cli_arguments() -> ArgumentParser:
     cache_help = "seconds how long requests should be cached (pass 0 to bypass the cache, -1 to cache forever)"
     parser.add_argument("--cache-duration", "-d", default=3600, type=int, help=cache_help, metavar="SEC")
 
+    parser.add_argument(
+        "--sort",
+        "-s",
+        help="Sorting method",
+        choices=["a", "alphabetic", "u", "updated"],
+        default="updated",
+        dest="sort",
+        const="updated",  # noqa: SC200
+        nargs="?",
+        metavar="[a(lphabetic), u(pdated)]",
```

Comment:
shouldn't this just be choice?

Reply:
I just though this: `--sort [{a,alphabetic,u,updated}], -s [{a,alphabetic,u,updated}]
                                   sorting method (default: updated)` looked a bit weird.
your choice though, let me know and I'll remove it

Reply:
I prefer without, so that would be great 👍